src/server/api/utility.ts

```
@@ -0,0 +1,1311 @@
+  if (!deconstructArray) {
+    return okAlias((s, t) => ({
+      status: s.Success,
+      type: t.Success,
+      data: resultArray as D extends true ? T : T[],
+    }));
+  }
+
+  const result = resultArray?.[0];
+  if (!result || result === undefined) {
+    switch (interactionType) {
+      default:
```

Biome (1.9.4)

[error] The default clause should be the last switch clause.

The following case clause is here:

Regardless its position, the default clause is always executed when there is no match. To avoid confusion, the default clause should be the last switch clause.

(lint/suspicious/useDefaultSwitchClauseLast)